src/CoDa.jl

```diff
@@ -20,7 +20,7 @@ using Printf
 import Tables
 import ScientificTypes as ST
 import TableTransforms as TT
-import TableTransforms: Transform, Stateless
+import TableTransforms: Transform, Stateless, Reject
```

Comment:
We are using Reject somewhere?

Reply:
Oh I see, you're using it in the revert call. I will remove the explicit import and will use `TT.Reject` instead.